src/app/routes/utils/imageUrl/index.js

```diff
@@ -0,0 +1,11 @@
+import is from 'ramda/src/is';
+
+export const FALLBACK_PLACEHOLDER_IMAGE_URL = `${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN}${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH}images/media_placeholder.png`;
+
+export const getPlaceholderImageUrl = imageUrl => {
```

Comment:
We try to give the folder the same name as the function - would it be possible to rename to ` src/app/routes/utils/getPlaceholderImageUrl/index.js` instead?